manifests/registry/component/backstage/deployment.yaml

```diff
@@ -37,10 +37,12 @@ spec:
             value: "backstage"
           - name: POSTGRES_PASSWORD
             value: "pass"
-          - name: APP_CONFIG_backend_baseUrl
```

Comment:
Oh, now I see how [this](https://github.com/Apicurio/backstage/pull/8#discussion_r1232518917) was working!

Reply:
Added in PR https://github.com/Apicurio/backstage/pull/12 thanks @andreaTP